docs/app/themes.ts

```diff
@@ -0,0 +1,2 @@
+import '@marigold/theme-b2b/styles.css';
```

Comment:
Delete, because unused. Sorry I added that :D

Reply:
Delete the file?

Reply:
yep